build/bin/release

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+if [ "$#" -lt "1" ]; then
+    echo "must specify chart directory"
+    exit 1
+fi
+
+set -x
+set -e
+
+chart=$1
+
+rm -r build/release
+
+owner=${GITHUB_USER:-onosproject}
+repo=${GITHUB_REPO:-onos-helm-charts}
+remote=${GIT_REMOTE:-origin}
+token=${GITHUB_TOKEN}
+branch=$(git branch --show-current)
```

Comment:
`git branch --show-current` is not a valid command on my git installation (v2.17.1 on Ubuntu 18.04)

`git rev-parse --abbrev-ref HEAD` works for me

Reply:
Interesting. Good catch